satpy/tests/writer_tests/test_cf.py

```diff
@@ -1225,93 +1226,111 @@ def test_collect_cf_dataarrays_with_latitude_named_lat(self, *mocks):
-        self.assertDictEqual(enc, {'y': {'_FillValue': None},
-                                   'x': {'_FillValue': None},
-                                   'lon': {'chunksizes': (2, 2)},
-                                   'foo': {'chunksizes': (2, 2)},
-                                   'bar': {'chunksizes': (2, 2)}})
+        expected_dict = {
+            'y': {'_FillValue': None},
+            'x': {'_FillValue': None},
+            'lon': {'chunksizes': (2, 2)},
+            'foo': {'chunksizes': (2, 2)},
+            'bar': {'chunksizes': (2, 2)}
+        }
+        assert enc == expected_dict
 
-    def test_with_time(self):
+    def test_with_time(self, fake_ds):
         """Test data with a time dimension."""
         from satpy.writers.cf_writer import update_encoding
 
         # With time dimension
-        ds = self.ds.chunk(8).expand_dims({'time': [datetime(2009, 7, 1, 12, 15)]})
+        ds = fake_ds.chunk(8).expand_dims({'time': [datetime(2009, 7, 1, 12, 15)]})
         kwargs = {'encoding': {'bar': {'chunksizes': (1, 1, 1)}},
                   'other': 'kwargs'}
         enc, other_kwargs = update_encoding(ds, kwargs)
-        self.assertDictEqual(enc, {'y': {'_FillValue': None},
-                                   'x': {'_FillValue': None},
-                                   'lon': {'chunksizes': (2, 2)},
-                                   'foo': {'chunksizes': (1, 2, 2)},
-                                   'bar': {'chunksizes': (1, 1, 1)},
-                                   'time': {'_FillValue': None,
-                                            'calendar': 'proleptic_gregorian',
-                                            'units': 'days since 2009-07-01 12:15:00'},
-                                   'time_bnds': {'_FillValue': None,
-                                                 'calendar': 'proleptic_gregorian',
-                                                 'units': 'days since 2009-07-01 12:15:00'}})
-
+        expected_dict = {
+            'y': {'_FillValue': None},
+            'x': {'_FillValue': None},
+            'lon': {'chunksizes': (2, 2)},
+            'foo': {'chunksizes': (1, 2, 2)},
+            'bar': {'chunksizes': (1, 1, 1)},
+            'time': {'_FillValue': None,
+                     'calendar': 'proleptic_gregorian',
+                     'units': 'days since 2009-07-01 12:15:00'},
+            'time_bnds': {'_FillValue': None,
+                          'calendar': 'proleptic_gregorian',
+                          'units': 'days since 2009-07-01 12:15:00'}
+        }
+        assert enc == expected_dict
         # User-defined encoding may not be altered
-        self.assertDictEqual(kwargs['encoding'], {'bar': {'chunksizes': (1, 1, 1)}})
+        assert kwargs['encoding'] == {'bar': {'chunksizes': (1, 1, 1)}}
 
 
 class TestEncodingKwarg:
     """Test CF writer with 'encoding' keyword argument."""
 
-    @pytest.fixture
+    @ pytest.fixture
```

Comment:
Is it correct that the precommit / autopep8 add a space here?

Reply:
No...it shouldn't. But Satpy also doesn't have any auto-formatting enabled in our pre-commit (I have some pytroll packages using black, but not satpy). Only imports should be updated. Are you sure this was pre-commit that did this? Do you have pre-commit changes in your local clone?

Reply:
It messed up with another precommit ...